appveyor.yml

```diff
@@ -1,5 +1,6 @@
 environment:
   matrix:
+    - node_version: "9"
```

Comment:
Since Node 9 will be "experimental" I don't think adding it to the build chain would be a good idea. Things may break or change randomly and that would cause churn with the team and reduce the signal/noise ratio for CI build results.

Reply:
I'm curios, what do you mean with experimental?

Reply:
Formally it's stable, not experimental. Experimental were RCs before the final. Node 8 is LTS but that doesn't mean Node 9 is not stable; it officially is.

Reply:
It's odd to test on node 5, which has been EOL for over a year, but not node current